src/document.h

```diff
@@ -140,6 +140,13 @@ typedef struct {
   RedisModuleString *languageStr;  // Language string for HSET
 } AddDocumentOptions;
 
+// When indexing the document we are okay with it being expired, saves us the work of indexing it
```

Comment:
Notice that the key isn't actually expired due to the `NOEFFECTS` flag. We just do not read it.